docs/cli-user-interface/markdown/README.md

```diff
@@ -2,9 +2,9 @@
 
 This documentation is auto-generated from `k8s-kms-plugin`:
 
-- version `v0.6.0-alpha-14-g2c6222b-dirty`
-- commit `2c6222bde64053b12f739af58c1cee97c32863ec`
-- build date 2025-07-31T09:35:04+00:00.
+- version ``
+- commit ``
+- build date .
```

Comment:
Version & commit info are missing, this is caused by missing `LDFLAGS` values when running `@go run cmd/k8s-kms-plugin/main.go docs ...` command in the Makefile compared to running this `docs` from the k8s-kms-plugin's binary.

One reason behind including the "version", "date" and "commit ID" in the auto-generated doc is to be able to know _for sure_ to which version of the tool this doc corresponds.

Drawback of this method is that the commit ID referenced here is obviously an older commit, since you need a new commit to commit the documentation.

We could consider a new way to populate those fields. This requires changes in the cobra command `cmd/k8s-kms-plugin/cmd/docs.go`.